### ISSUE TITLE
"duplicacy list -chunks" should list chunk sizes

### ISSUE THREAD

**Issue by fracai** [gilbertchen/duplicacy#178]
It'd be useful to see the chunk sizes in the output because it would allow calculating additional revision stats.

Specifically, I created a python script that reports the number of chunks that are shared by each revision pair and what is added and unique to each revision. Knowing the sizes that are (I think) available in the snapshots files would allow calculating the amount of space taken up by each revision and how much can be freed by pruning any given revision.

Ultimately I think it would make sense to roll these stats in to duplicacy itself.
- Prune could include a dry run mode that could report how much space will be freed for a given set of -keep arguments.
- List could report the unique and new sizes. And the size of each revision.
- Copy could report how much data will be transferred.

[rev-stats.py](https://github.com/gilbertchen/duplicacy/files/1291147/rev-stats.py.txt)
[Output example](https://github.com/gilbertchen/duplicacy/files/1291146/rev-stats.txt)

**Comment by fracai** [comment]
I can see now that the "check -stats" action actually displays some of the stats that I'm interested in.

**Comment by gilbertchen** [comment]
Right.  I think if we add more stats like what you suggested it should go into the check command.

**Comment by TheBestPessimist** [comment]
@gilbertchen may we close this issue?
